### PR TITLE
doc: notes for 2020-01-08

### PR DESCRIPTION
**Pull request by robpalme**
Refs: https://github.com/nodejs/modules/issues/461
Refs: https://docs.google.com/document/d/1RwdeqHV9oJBBafcAajWZQ0eCnzcMNSo-dQGN1WeEszY/
Closes: https://github.com/nodejs/modules/issues/461

### CODE REVIEW

File: doc/meetings/2020-01-08.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+* BF: Please confirm the previous questions about JSON order and relation to ecosystem tooling.  It's a well-defined ordering.  We have found no ecosystem issues.  `OrdinaryOwnPropertyKeys` is the official algorithm.
+* GB: Gus requested array indexes do not follow the same ordering.  Numbers come first, then insertion ordering. So userland tooling could get this wrong.  The PR addresses this by adding validation, to ensure there are NO array indexes in the object, otherwise throw an error.  It's fully well-defined and aligned with the spec. Node achieved consensus on this. I feel `exports` is now feature complete.  It's really nice. The only outstanding PR is unflagging.
```
Comment (ljharb) [inline doc/meetings/2020-01-08.md]: re: node achieved consensus on this. was this ever discussed in a modules meeting tho prior to landing it? even though that's not a requirement, that seems like an appropriate/courteous thing to do.

File: doc/meetings/2020-01-08.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+
+#### module: unflag resolve self [#31002](https://github.com/nodejs/node/pull/31002)
+
+* MB: Landed. Released in 13.6.
+* JK: Congrats to Guy Bedford for making this happen.
+* GB: It's backwards compatible with CJS. We compromised by saying the feature is only active when `exports` exists in `package.json`. It's another justification for preferring `exports`.
```
Comment (ljharb) [inline doc/meetings/2020-01-08.md]: was there any modules group discussion about this compromise in a meeting?

Comment (GeoffreyBooth) [reply doc/meetings/2020-01-08.md]: Yes, in the last meeting I think.

File: doc/meetings/2020-01-08.md (hunk 1 above)
Comment (GeoffreyBooth) [reply doc/meetings/2020-01-08.md]: Yes, it was discussed in 2019-12-18: https://github.com/nodejs/modules/issues/455, https://docs.google.com/document/d/107-EWDmaUN5XCHNzme6h5AYfl1L3J-ZbgZSq7bqWC8E/edit

File: doc/meetings/2020-01-08.md (hunk 2 above)
Comment (ljharb) [reply doc/meetings/2020-01-08.md]: thanks, that's on me then for missing that day.

File: doc/meetings/2020-01-08.md (hunk 1 above)
Comment (ljharb) [reply doc/meetings/2020-01-08.md]: thanks, that's on me then for missing that day.

File: doc/meetings/2020-01-08.md (hunk 2 above)
Comment (GeoffreyBooth) [reply doc/meetings/2020-01-08.md]: BTW big shout out to @robpalme for taking great detailed notes!

**Pull request merged**